packages/lit-html/package.json

```diff
@@ -57,10 +61,14 @@
     "/lit-html.d.ts.map",
     "/lit-html.js",
     "/lit-html.js.map",
-    "/parts.d.ts",
-    "/parts.d.ts.map",
-    "/parts.js",
-    "/parts.js.map",
+    "/directive.d.ts",
+    "/directive.d.ts.map",
+    "/directive.js",
+    "/directive.js.map",
+    "/directive-helpers.d.ts",
+    "/directive-helpers.d.ts.map",
+    "/directive-helpers.js",
+    "/directive-helpers.js.map",
```

Comment:
Sheesh. And `disconnectable-directive` here too 🙏

Reply:
done